server/pkg/schema/schema.go

```diff
@@ -180,3 +186,186 @@ func (s *Schema) CopyFrom(s2 *Schema) {
 	s.fields = slices.Clone(s2.fields)
 	s.titleField = s2.TitleField().CloneRef()
 }
+
+type GuessFieldData struct {
+	SchemaID id.SchemaID
+	Type     value.Type
+	Name     string
+	Key      string
+}
+
+func (s *Schema) GuessSchemaFieldFromAssetFile(file io.ReadCloser, isGeoJSON bool) ([]GuessFieldData, error) {
```

Comment:
Test is missing!
We should use this implementation in the item import functionality.